opencolorio_config_aces/clf/transforms/arri/generate.py

```diff
@@ -0,0 +1,445 @@
+        linSideBreak=linSideBreak,
+        logSideSlope=logSideSlope,
+        logSideOffset=logSideOffset,
+        linSideSlope=linSideSlope,
+        linSideOffset=linSideOffset,
+        direction=direction,
+    )
+
+
+def _build_awg4_mtx():
+    """
+    Build the `MatrixTransform` for ARRI Wide Gamut 4 primaries.
+
+    Values are copied directly from the LogC4 Specification.
+
+    Returns
+    -------
+    ocio.MatrixTransform
+         *OpenColorIO* `MatrixTransform`.
+    """
+
+    # Values as defined in: "ARRI LogC4 Logarithmic Color Space Specification".
+    # Extra row and column for OCIO alpha channel.
+    mtx = [
+        0.750957362824734131,
+        0.144422786709757084,
+        0.104619850465508965,
+        0.0,
+        0.000821837079380207,
+        1.007397584885003194,
+        -0.008219421964383583,
+        0.0,
+        -0.000499952143533471,
+        -0.000854177231436971,
+        1.001354129374970370,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        1.0,
+    ]
+
+    return transform_factory(transform_type="MatrixTransform", matrix=mtx)
+
+
+def _build_logc4_curve():
+    """
+    Build the `LogCameraTransform` for ARRI LogC4 Curve.
+
+    Parameter values are derived from the published LogC4 Specification.
+
+    Returns
+    -------
+    ocio.LogCameraTransform
+         *OpenColorIO* `LogCameraTransform`.
+    """
+
+    # Parameters as defined in "ARRI LogC4 Logarithmic Color Space Specification"
+    # Parameters as defined in aces-dev CTL
+    a = (2.0**18.0 - 16.0) / 117.45
+    b = (1023.0 - 95.0) / 1023.0
+    c = 95.0 / 1023.0
+    t = (pow(2.0, 14.0 * (-c / b) + 6.0) - 64.0) / a
+
+    # OCIO LogCameraTransform translation variables
+    linSideSlope = [a] * 3
+    linSideOffset = [64.0] * 3
+    logSideSlope = [b / 14.0] * 3
+    logSideOffset = [-6.0 * b / 14.0 + c] * 3
+    linSideBreak = [t] * 3
+    base = 2.0
+    direction = ocio.TRANSFORM_DIR_INVERSE
+
+    return transform_factory(
+        transform_type="LogCameraTransform",
+        transform_factory="Constructor",
+        base=base,
+        linSideBreak=linSideBreak,
+        logSideSlope=logSideSlope,
+        logSideOffset=logSideOffset,
+        linSideSlope=linSideSlope,
+        linSideOffset=linSideOffset,
+        direction=direction,
+    )
+
+
+def _generate_logc3_transforms(
+    transforms_set, output_directory, ei_list=[800]
+):
+    """
+    Generate the collection LogC3 of transforms.
+
+    Parameters
+    ----------
+    transforms_set : dict
+        A `dict` of transforms to append to.
+    output_directory : PosixPath or WindowsPath
+        Directory where generated CTL files should be saved.
+    ei_list : array_like of int, optional
+        List of EI values to generate LogC3 and LogC3 Curve transforms for.
+
+    Returns
+    -------
+    dict
+        Dictionary of *OpenColorIO* `Transform` instances.
+    """
+
+    for ei in ei_list:
+
+        # Genereate ARRI LogC3 to ACES 2065-1 Transform
```

Comment:
Genereate --> Generate
Same below (do a search and replace).

Reply:
Typo. Fixed.